src/components/Pagination.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import classnames from "classnames";
+import "../styles/Pagination.scss";
+
+const range = (from, to) => {
+  let length = to - from + 1;
+  return Array.from({length: length}, (_, idx) => idx + from);
+}
+
+const DOTS = "...";
+
+class Pagination extends React.Component {
+  
+  handleLeft = () => {
+    this.props.onPageChange(this.props.currentPage - 1);
+  }
+
+  handleRight = () => {
+    this.props.onPageChange(this.props.currentPage + 1);
+  }
+
+  fetchPageNumbers = () => {
+    const totalPages = Math.ceil(this.props.total / this.props.pageSize);
+    const {currentPage} = this.props;
+    const siblings = 1;
+
+    const totalNumbers = siblings + 5;
```

Comment:
it's not very intuitive what totalnumbers mean

Reply:
added comments explaining this